### PR TITLE
Maintenance: Update Sorting and grouping wording

### PR DESCRIPTION
**Pull request by MrGeneration**

### CODE REVIEW

File: manage/overviews.rst [hunk 1]
```diff
@@ -108,15 +108,26 @@ Attributes
...
-     attribute within the list?
-   - active: Set them active or inactive
+   :Sorting by:
+      In which order should the tickets be displayed? (Sorted by the attributes)
+
+   :Sorting order:
+      The direction of the sorting.
+
+   :Grouping by:
+      Should the tickets be displayed again grouped by a specific attribute
```
Comment (mgruner) [inline manage/overviews.rst:121]: Should the displayed tickets be grouped by a specific attribute within the list?

File: manage/overviews.rst [hunk 2]
```diff
@@ -108,15 +108,26 @@ Attributes
...
+      within the list?
+
+   :Grouping order:
+      The direction of the grouping.
+
+   :Active:
+      Should the overview be active or not?
+      Inactive overviews are not available to your users but are an alternative
```
Comment (mgruner) [inline manage/overviews.rst:129]: Rather than deleting an overview entirely, you can set it to inactive to make it unavailable to your users.

**Review by mgruner — APPROVED** [review]
Looks great, just minor style/grammar suggestions.

File: manage/overviews.rst (hunk 1 above)
Comment (MrGeneration) [reply manage/overviews.rst:121]: Oops my bad, some one was sluggish there.

**Pull request merged**